test-ts/utils.ts

```diff
@@ -311,7 +311,7 @@ export function getFirstBlockNumberForEpoch(
 
 // `useGanache` allows for mining block directly on the ganache network
 export async function mineBlocks(blocks: number, useGanache?: boolean) {
-  let localProvider: JsonRpcProvider;
+  let localProvider: any;
```

Comment:
This can be removed once we use latest hardhat

Reply:
Would probably be good to add a `//TODO:` here so that the comment is persistent once merged.